datasette_graphql/utils.py

```diff
@@ -597,12 +581,13 @@ async def resolve_table(
                     path_with_query_string,
                 )
 
-        request = Request.fake(path_with_query_string)
-
-        view = TableView(DatasetteSpecialConfig(datasette))
-        data, _, _ = await view.data(
-            request, database=database_name, hash=None, table=table_name, _next=after
-        )
+        data = (await datasette.client.get(path_with_query_string)).json()
+        data["rows"] = [dict(zip(data["columns"], row)) for row in data["rows"]]
+        # If any cells are $base64, decode them into bytes objects
+        for row in data["rows"]:
+            for key, value in row.items():
+                if isinstance(value, dict) and value.get("$base64"):
+                    row[key] = b64decode(value["encoded"])
```

Comment:
This block feels a little clumsy - I'm messing around with `data["rows"]` to compensate for a difference in shape between the JSON and the raw data that came back from `view.data(...)`.

Reply:
One complexity here is that I'd like to get back dictionaries, rather than an array of arrays that I have to `zip()` against the columns - but I also need the other stuff in `data` - in particular here: https://github.com/simonw/datasette-graphql/blob/a4f8a009cc392065b7dcd3e8368b031a38402707/datasette_graphql/utils.py#L263-L264

Reply:
So maybe this should be blocked on changes to Datasette's default JSON? https://github.com/simonw/datasette/issues/782